infrastructure/helm/keycloak/keycloak-setup.md

```diff
@@ -0,0 +1,64 @@
+# Install Quantumserverless and Configure Keycloak
+## In the command line
+Switch default namespace to the install target namespace
+```
+kubectl config set-context –current –namespace="target namespace"
+```
+Install helm chart
+```
+helm install "release name" quantemserverless
+```
+Wati for all pods come up except `releasename-kuberay-head-xxxx`
+Run ./login-setup.sh <RELEASE NAME>
+
+## In the browser
+# Configure Keycloak
+
+Open browser at http://<LOCALIP>:<KEYCLOAK PORT>
+ - Select Administration Console
+ - Login with “user” / <PASSWORD>
+  <LOCALIP>, <KEYCLOAK PORT>, <PASSWORD> are in the login-setup.sh script output.
+
+Create “quantumserverless” Realm
+ - Dropdown “master”
+  -- “Create Realm”
+  -- Set “Realm name” “quantumserverless”
+  -- “Create”
+ - "Realm settings" in the left Nav
+  -- "Action"
+  -- "Partial import"
+  -- "Browse"
+  -- Select "keycloakconfig.json"
+  -- Check all resource check boxes and select Overwrite
+  -- "Import"
+  -- "Close"
+
+Generate “rayclient” client secret
+ - “Clients” in the left nav
+  -- Select "rayclient"
+  -- “Credentials” tab
+  --"Regenerate" the "Client Secret"
+  -- "Yes"
+  -- Take a Note “Client secret” value
+
+Create “admin” user
```

Comment:
Can this be done through configuration or environment variables like the another admin user?

Reply:
So far I couldn't find a way to do that.  I'll keep looking for it.

Reply:
I'm going to try to investigate a little bit more about this 🙏